release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -298,6 +298,91 @@ With this release, the `MaxUnavailableStatefulSet` featureset configuration para
+** Modified the `NodeHighNumberConntrackEntriesUsed`, `NodeNetworkReceiveErrs` and `NodeNetworkTransmitErrs` alerts to evaluate only metrics from the `node-exporter` job.
+
+[id="ocp-4-14-monitoring-new-option-to-create-alerts-based-on-core-platform-metrics"]
+==== New option to create alerts based on core platform metrics
+
+With this release, administrators can create new alerting rules based on core platform metrics.
+You can now modify settings for existing platform alerting rules by adjusting thresholds and by changing labels.
+You can also define and add new custom alerting rules by constructing a query expression based on core platform metrics in the `openshift-monitoring` namespace.
+This feature was included as a technology preview in a previous {product-title} release and is now generally available.
+For more information, see xref:../monitoring/managing-alerts.adoc#managing-core-platform-alerting-rules_managing-alerts[Managing alerting rules for core platform monitoring].
+
+[id="ocp-4-14-monitoring-new-option-to-specify-resource-limits-for-all-monitoring-components"]
+==== New option to specify resource limits for all monitoring components
+
+With this release, you can now specify resource requests and limits for all monitoring components, including the following:
+
+* Alertmanager
+* kube-state-metrics
+* monitoring-plugin
+* node-exporter
+* openshift-state-metrics
+* Prometheus
+* Prometheus Adapter
+* Prometheus Operator and its admission webhook service
+* Telemeter Client
+* Thanos Querier
+* Thanos Ruler
+
+In previous versions of {product-title}, you could only set options for Prometheus, Alertmanager, Thanos Querier, and Thanos Ruler.
+
+[id="ocp-4-14-monitoring-new-options-to-configure-node-exporter-collectors"]
+==== New options to configure node-exporter collectors
+
+With this release, you can customize Cluster Monitoring Operator (CMO) config map settings for additional node-exporter collectors.
+The following node-exporter collectors are now optional, and you can enable or disable each one individually by using config map settings:
+
+* `ksmd` collector
+* `mountstats` collector
+* `processes_linux` collector
+
+In addition, you can now exclude network devices from the relevant collector configuration for the `netdev` and `netclass` collectors.
+
+[id="ocp-4-14-monitoring-new-options-to-deploy-monitoring-web-console-plugin-resources"]
+==== New option to deploy monitoring web console plugin resources
+
+With this release, the monitoring pages in the *Observe* section of the {product-title} web console are deployed as a xref:../web_console/dynamic-plugin/overview-dynamic-plugin.adoc[dynamic plugin].
+With this change, the Cluster Monitoring Operator (CMO) is now the component that deploys the {product-title} web console monitoring plugin resources.
+You can now use CMO settings to configure the following features of the console monitoring plugin resource:
+
+* Node selectors
+* Node tolerations
```

Comment:
I think the terminology is just `Tolerations` 
```suggestion
* Tolerations
```

Reply:
Changed per your suggestion